### DISCUSSION TITLE
Blue spinning wheel

### DISCUSSION THREAD

**Discussion by jeffprandall** [Budibase/budibase#8707] [Help]
I updated my self hosted instance using the standard docker-compose file to version 2.1.28 but on any app when I get to the Design screen I get a blue spinning wheel.  I have tried in Chrome 107.0.5304.107, Firefox 107.0 (64-bit) and Edge Version 107.0.1418.42.

![image](https://user-images.githubusercontent.com/119567/202329494-74ef3350-2533-48cf-bcf5-7d4b66299b24.png)

**Comment by andz-bb** [comment]
Hey @jeffprandall - could you check your browser's console for any errors when recreating the spinning wheel? You'll be able to see it in developer tools by pressing F12 in your browser. That will help us to identify what's going wrong when the builder gets stuck. Thanks

**Reply by jeffprandall** [reply]
I don't see any errors in the console.  And on the network tab all the resources come back with a 200 status code. 

![image](https://user-images.githubusercontent.com/119567/202479015-42d2b21c-698e-4850-aaa7-9dc6be6f4bd2.png)

**Reply by andz-bb** [reply]
ok, thanks for checking @jeffprandall - could you send an app export to support@budibase.com or on our discord server so that I can do some investigation on my end? thanks

**Reply by jeffprandall** [reply]
I posted in the Discord channel

**Comment by tbox98** [comment]
I have the same issue after updating to version 2.1.28. Tried with Safari, Chrome and Firefox.

**Reply by jeffprandall** [reply]
I replaced my old docker compose file with the new one posted here `https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml` and it started working.  I didn't get a chance to compare my old one to this one to see the difference.

**Reply by tbox98** [reply]
We tried your solution, but it didn't work. We get the following error message from Chrome console: Refused to display 'https://ourapp.com/' in a frame because it set multiple 'X-Frame-Options' headers with conflicting values ('SAMEORIGIN, DENY'). Falling back to 'deny'.

**Reply by tbox98** [reply]
UPDATE: We solved the problem by changing the settings of the nginx server regarding SSL connections.
